extensions.json

```diff
@@ -1147,6 +1147,11 @@
       "id": "oderwat.indent-rainbow",
       "repository": "https://github.com/oderwat/vscode-indent-rainbow"
     },
+    {
+      "id": "oguhpereira.spotify-color-theme",
+      "repository": "https://github.com/oguhpereira/vscode-spotify-theme",
+      "version": "1.2.9"
```

Comment:
```suggestion
      "version": "1.2.9",
      "checkout": "v1.2.9"
```

Reply:
> Thanks! When a tag is available for the version to be installed, I think we should use it.

@spoenemann I just wanna note that I prefer to use `npm run add`, it requires less time to add a new extension, I added this one by using command below:
```bash
$  npm run add -- https://github.com/oguhpereira/vscode-spotify-theme
```

It looks like we can improve `add-extension.js` util to find the corresponding tag and use it instead of doing manual steps for this.

What do you think about it?